lib/cisco_node_utils/cmd_ref/DEPRECATED.yaml

```diff
@@ -0,0 +1,95 @@
+# DEPRECATED
```

Comment:
It's a common deprecation file. We're going to put the cisco_vlan deprecated yaml entries in there as well. There's no need to separate them.